src/main/java/fruit/shop/service/parser/RecordsParser.java

```diff
@@ -0,0 +1,7 @@
+package fruit.shop.service.parser;

```

Comment:
It is not necessary to create new subpackage if there only one class, fix every where
```suggestion
package fruit.shop.service;
```